src/Header/style.scss

```diff
@@ -0,0 +1,7 @@
+.container {
```

Comment:
is this general class?

Reply:
no it just for promo and navigation bar ,so i changed name to more understandable